subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -4493,14 +4513,17 @@ static inline int length_req_rsp_recv(struct ll_conn *conn, memq_link_t *link,
 #if defined(CONFIG_BT_CTLR_PHY)
 		u16_t max_rx_time;
 		u16_t max_tx_time;
+		u16_t temp_rx, temp_tx;
+
+		dle_max_time_get(conn, &temp_rx, &temp_tx);
+		max_rx_time = sys_le16_to_cpu(temp_rx);
+		max_tx_time = sys_le16_to_cpu(temp_tx);
```

Comment:
This looks incorrect. Why would you endianness-convert from LE, when value is provided by dle_max_time_get?

Reply:
as per suggestion from Vinayak; we were seeing some issues trying to resolve

Reply:
removed